templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -77,6 +77,32 @@ contents:
           mountPath: /etc/ssl/etcd/
         - name: sa
           mountPath: /var/run/secrets/kubernetes.io/serviceaccount/
+      - name: wait-for-membership
+        image: ""{{.Images.etcdKey}}""
+        command:
+        - /bin/sh
+        - -c
+        - |
+          #!/bin/sh
+          set -euxo pipefail
+
+          source /run/etcd/environment
+
+          export ETCDCTL_API=3 ETCDCTL_CACERT=/etc/ssl/etcd/ca.crt ETCDCTL_CERT=$(find /etc/ssl/ -name *peer*crt) ETCDCTL_KEY=$(find /etc/ssl/ -name *peer*key)
+
+          until $(etcdctl member list | grep $ETCD_DNS_NAME>>/dev/null)
```

Comment:
note: if you want to send command output to /dev/null you can use a single `>` operator